backends/arm/arm_quantizer_utils.py

```diff
@@ -780,11 +780,11 @@ def _annotate_add(
         if _is_annotated([add_node]):
             continue
 
+        input_act0 = add_node.args[0]
         input_act_qspec = get_input_act_qspec(quantization_config)
-        output_act_qspec = get_output_act_qspec(quantization_config)
+        shared_with_input0_qspec = SharedQuantizationSpec((input_act0, add_node))
```

Comment:
Curious why are we doing this?

Reply:
To get the same quantization parameters to be used on the operands and the result of the add in order to do some optimization in Vela.